src-test/src/org/openbravo/test/costing/TestCostingBase.java

```diff
@@ -80,6 +83,33 @@ public void setInitialConfiguration() {
         organization.setCurrency(OBDal.getInstance().get(Currency.class, EURO_ID));
         OBDal.getInstance().save(organization);
 
+        // Create Internal Consumption Document Type if it does not exist for this context
+        OBCriteria<DocumentType> internalConsCrit = OBDal.getInstance().createCriteria(DocumentType.class);
```

Comment:
If you will be use an hardcored client and organization IDs you must set this on you criterial:

    internalConsCrit.setFilterOnReadableClients(false);
    internalConsCrit.setFilterOnReadableOrganization(false);

Reply:
It's not actually needed to specify the client and organization, so I took these constraints out. The tests still pass